src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -54,9 +54,9 @@ internal class HttpTriggerAttributeBindingProvider : ITriggerBindingProvider
             typeof(HttpRequestMessage)
         };
 
-        public HttpTriggerAttributeBindingProvider(Action<HttpRequest, object> responseHook)
+        public HttpTriggerAttributeBindingProvider(HttpOptions options)
         {
-            _responseHook = responseHook;
+            _options = options;
```

Comment:
Was your intention to take a dependency on `IOptions<HttpOptions>` instead? 

Can you also ensure this is not null?